spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User pages test index/show ', type: :feature do
+  before(:all) do
+    @first_user = User.create(name: 'Solomon', photo: 'https://avatars.githubusercontent.com/u/101924220?v=4',
+                              bio: 'test for User')
+    @post1 = Post.create(title: 'Rspec test 1', text: 'rspec test for post', user_id: @first_user)
```

Comment:
@Maxwell011 You did a great job! We appreciate your hard work. However, we still have failing tests. I hope I would be able to describe them easily to you. Kindly see the image we have:

<img width="1074" alt="Screenshot 2023-02-16 at 00 14 56" src="https://user-images.githubusercontent.com/69011963/219130017-747c7b76-092b-4992-80cf-eb2f45b91e3e.png">
<img width="1172" alt="Screenshot 2023-02-16 at 00 15 06" src="https://user-images.githubusercontent.com/69011963/219130040-3ada7277-06c6-4d65-bbaf-593b6f6006c9.png">

As you can see in these images only these five have the exact same issue. I think you wanted to make author_id instead of user_id. Because you specified your user as the author. I hope this message was helpful to you!

- [ ] Make sure all tests are passing. I suggest you run `rails db: reset` before retesting your app to clear your old data and replace it with new ones in seeds.